docs/static/monitoring-apis.asciidoc

```diff
@@ -0,0 +1,312 @@
+[[monitoring]]
+== Monitoring APIs
+
+Logstash provides the following monitoring APIs to retrieve runtime metrics
+about Logstash:
+
+* <<hot-threads-api>>
+* <<root-resource-api>>
+* <<stats-info-api>>
+
+//NOTE: Need to add this to the doc after Alpha 1: * <<plugins-api>>
+
+=== Testing Logstash APIs in Sense
+
+//QUESTION: What do you think of this approach? I felt that the links for invoking Sense would be useful, but I wanted to avoid confusing users. If they don't notice this topic, they might still be confused. Do you think I should keep the links that launch Sense, or should I remove them?
+
+Sense is a Kibana app that you can use to build and test REST calls. You'll notice
+that the monitoring API documentation includes links for viewing the examples
+in Sense. For example:
+
+[source,js]
+--------------------------------------------------
+GET /
+--------------------------------------------------
+//AUTOSENSE
+
+After clicking the link to view the example, you need to change the address
+in the Server field to point to port `9600` before you can successfully test the
+monitoring APIs. 
+
+For example, if Kibana is running on localhost, you need to change the value
+of the Server field from `http://localhost:9200` to `http://localhost:9600`.
+
+[[monitoring-common-options]]
+=== Common Options
+
+The following options can be applied to all of the Logstash monitoring APIs.
+
+[float]
+=== Pretty Results
+
+When appending `?pretty=true` to any request made, the JSON returned
+will be pretty formatted (use it for debugging only!). Another option is
+to set `?format=yaml` which will cause the result to be returned in the
+(sometimes) more readable yaml format.
+
+[float]
+=== Human-Readable Output
+
+Statistics are returned in a format suitable for humans
+(eg `"exists_time": "1h"` or `"size": "1kb"`) and for computers
+(eg `"exists_time_in_millis": 3600000` or `"size_in_bytes": 1024`).
+The human-readable values can be turned off by adding `?human=false`
+to the query string. This makes sense when the stats results are
+being consumed by a monitoring tool, rather than intended for human
+consumption.  The default for the `human` flag is
+`false`.
```

Comment:
> Are you planning to add wider support for ?human post-Alpha1? If so, I'm wondering if it would make sense to document the behavior as we'll deliver it in 5.0.0 and add an Alpha1 note indicating that the parameter is only supported for the hot threads API.

I think this would be a great idea, since we will probably add more support to the `human` option.